cypress/support/commands.js

```diff
@@ -22,4 +22,15 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('createWorker', (worker) => {
+  cy.get('#addNewRecordButton').click();
+  cy.get('#firstName').type(worker.firstName);
+  cy.get('#lastName').type(worker.lastName);
+  cy.get('#userEmail').type(worker.email);
+  cy.get('#age').type(worker.age);
+  cy.get('#salary').type(worker.salary);
+  cy.get('#department').type(worker.department);
+  cy.get('#submit').click();
```

Comment:
It's a good practice to validate form submission. You can add a check to ensure the worker was created successfully after clicking the submit button.